src/elements/HiddenChipsBlock/HiddenChipsBlock.js

```diff
@@ -88,15 +90,17 @@ const HiddenChipsBlock = ({
             }
           >
             <Chip
-              chipIndex={index + chipIndex}
               chip={element}
+              chipIndex={index + chipIndex}
```

Comment:
For string concatenation, please use string template literals instead of the `+`  opertaor.
```suggestion
              chipIndex={`${index}${chipIndex}`}
```